src/org/usfirst/frc/team694/robot/commands/auton/routines/SingleCubeSwitchAutonCommand.java

```diff
@@ -0,0 +1,29 @@
+package org.usfirst.frc.team694.robot.commands.auton.routines;
+
+import org.usfirst.frc.team694.robot.Robot;
+import org.usfirst.frc.team694.robot.commands.LiftMoveToHeightCommand;
+import org.usfirst.frc.team694.robot.commands.QuisitorDeacquireCommand;
+import org.usfirst.frc.team694.robot.commands.auton.DriveStraightPIDCommand;
+import org.usfirst.frc.team694.robot.commands.auton.DrivetrainDriveCurveCommand;
+import org.usfirst.frc.team694.robot.commands.auton.DrivetrainMoveInchesEncoderCommand;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class SingleCubeSwitchAutonCommand extends CommandGroup {
+
+    public SingleCubeSwitchAutonCommand() {
+        DrivetrainDriveCurveCommand driveCommand = new DrivetrainDriveCurveCommand(283);
+        
+        addParallel(new LiftMoveToHeightCommand(0.0)); //TODO: Find a height for the lift to move to.
```

Comment:
This shouldn't be zero. Find the height that we move the lift in master and use that one.

If we leave this at zero, the acquirer can break during testing